lib/iris/experimental/representation.py

```diff
@@ -291,23 +292,10 @@ def _make_content(self):
                         title = body.pop(0)
                         colspan = 0
                     elif k in self.two_cell_headers:
-                        try:
-                            split_point = line.index(":")
-                        except ValueError:
```

Comment:
Intending to lose this code altogether : see discussion.

Reply:
Great improvement